### PR TITLE
Update sumneko lua docs

### LINKED ISSUE #68: lua lsp docs should be expanded
Right now, trying to configure the lua lang-server is a bit difficult. It's not clear the best way to customize things based on someones machine/env.

### PR DESCRIPTION
**Pull request by mjlbach**
Closes #68?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4790,6 +4790,35 @@ require'lspconfig'.sumneko_lua.setup{
...
 
+The settings of the language server can also be overridden. This is especially useful
+when developing on the lua components of neovim. Some recommended settings are as follows:
+
+```lua
+require'lspconfig'.sumneko_lua.setup {
+  settings = {
+    Lua = {
+      runtime = {
+        -- Tell the language server which version of Lua you're using (LuaJIT in the case of Neovim)
```
Comment (teto) [inline README.md:4801]: It can be used with plain lua too so I would add a `most likely`.

File: README.md [hunk 2]
```diff
@@ -3190,7 +3190,7 @@ require'lspconfig'.omnisharp.setup{}
   Commands:
   
   Default Values:
-    cmd = { "/home/runner/.cache/nvim/lspconfig/omnisharp/run", "--languageserver", "--hostPID", "2524" }
+    cmd = { "/home/runner/.cache/nvim/lspconfig/omnisharp/run", "--languageserver", "--hostPID", "2517" }
```
Comment (teto) [inline README.md:3193]: this appears in all your PRs xD

Comment (mjlbach) [reply README.md:3193]: GHA docgen hates port 2524 for some reason🤷‍♀️

Comment (teto) [reply README.md:3193]: ha I see the command is non-stable `cmd = installer.info().cmd;` xD

**Comment by mjlbach** [comment]
also this is shamelessly stolen from the great https://github.com/nanotee/nvim-lua-guide

Thanks @nanotree

File: README.md (hunk 1 above)
Comment (mjlbach) [reply README.md]: addressed

**Pull request merged**